### PR TITLE
4534 AES Encryption Update based on skill case

### PR DESCRIPTION
**Pull request by ramkumar-k-9286**
AES Encryption Update based on skill case:

2 docs updated:

https://draft-openlibertyio.mybluemix.net/docs/21.0.0.9/password-encryption.html
https://draft-openlibertyio.mybluemix.net/docs/21.0.0.9/reference/command/securityUtility-encode.html

#4534

### CODE REVIEW

File: modules/reference/pages/command/securityUtility-encode.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ securityUtility encode [options]
 |--encoding=_encoding_type_
 |Specifies how to encode the password.
 Supported encodings types are `xor`, `aes`, and `hash`.
-If this option isn't specified, a default encoding type of `xor` is used.
+If this option isn't specified, a default encoding type of `xor` is used. Currently, aes type uses AES 128 encryption.
```
Comment (dmuelle) [inline modules/reference/pages/command/securityUtility-encode.adoc]: Currently, aes type uses AES 128 encryption.

---->

Currently, the `aes` type uses AES-128 encryption.

File: modules/ROOT/pages/password-encryption.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@
 = Password encryption limitations
 
 Open Liberty supports Advanced Encryption Standard (AES) encryption for passwords that are stored in the `server.xml` file.
-When you use this option for protecting passwords in the Open Liberty configuration, understand the limits to the protection that AES encryption provides.
+When you use this option for protecting passwords in the Open Liberty configuration, understand the limits to the protection that AES encryption provides. Currently, Open Liberty supports AES 128 encryption.
```
Comment (dmuelle) [inline modules/ROOT/pages/password-encryption.adoc]: AES-128

**Review by dmuelle — COMMENTED** [review]
two minor comments- disregard the other file changes in this PR- they are ok

**Comment by ramkumar-k-9286** [comment]
@dmuelle 
Correction done. Updated AES 128 to AES-128.

File: modules/reference/pages/command/securityUtility-encode.adoc [hunk 3]
```diff
@@ -55,7 +55,7 @@ securityUtility encode [options]
 |--encoding=_encoding_type_
 |Specifies how to encode the password.
 Supported encodings types are `xor`, `aes`, and `hash`.
-If this option isn't specified, a default encoding type of `xor` is used.
+If this option isn't specified, a default encoding type of `xor` is used. Currently, aes type uses AES-128 encryption.
```
Comment (dmuelle) [inline modules/reference/pages/command/securityUtility-encode.adoc]: Currently, aes type uses AES-128 encryption
---->
 Currently, the `aes` type uses AES-128 encryption

**Review by dmuelle — CHANGES_REQUESTED** [review]
monospace `aes` type (lowercase instance only)

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**